docker/docker-compose.yml

```diff
@@ -121,7 +121,7 @@ services:
       timeout: 5s
       retries: 5
   frontend-node-0:
-    image: "ghcr.io/singularity-data/risingwave:nightly-20220620"
+    image: "ghcr.io/singularity-data/risingwave:nightly-20220718"
```

Comment:
Don't change this. 20220718 nightly is not available. We should use an old version here.